src/ui/Querybox/Querybox.ts

```diff
@@ -220,11 +220,8 @@ export class Querybox extends Component {
     }
 
     if (!_.isEmpty(this.lastQuery)) {
-      if (!this.options.enableQuerySyntax) {
-        args.queryBuilder.expression.add("<@- " + this.lastQuery + " -@>");
-      } else {
-        args.queryBuilder.expression.add(this.lastQuery);
-      }
+      args.queryBuilder.disableQuerySyntax = !this.options.enableQuerySyntax;
+      args.queryBuilder.expression.add(this.lastQuery);

```

Comment:
Missing the check about whether or not the query is empty (adding an empty query in the query builder will break)